FrontPage/auth/AuthViewController.swift

```diff
@@ -212,4 +212,12 @@ extension AuthViewController {
     
   }
   
+  func getToken() -> String! {
+    return AuthViewController.self.token!
```

Comment:
Question/Suggestion: The AuthViewController has the `authState` property which allows us to access the `authState.lastTokenResponse?.accessToken` property. Do we need to do the `self.saveToken` stuff? Could we not simply return `authState.lastTokenResoinse?.accessToken` when `getToken` is called? It kinda seems like we're duplicating the token onto the AuthViewController class and I would be worried that we could end up with timing issues where the token is outdated in AuthViewController while the authState has the latest one.

Reply:
that's right, thanks Dara!